src/NuGet.Core/NuGet.Commands/Utility/AssetTargetFallbackUtility.cs

```diff
@@ -67,7 +67,7 @@ public static bool HasInvalidFallbackCombination(PackageSpec spec)
                 throw new ArgumentNullException(nameof(spec));
             }
 
-            return spec.TargetFrameworks.Any(e => e.Imports.Count > 0 && e.AssetTargetFallback.Count > 0);
+            return spec.TargetFrameworks.Any(e => e.PackageTargetFallbacks.Count > 0 && e.AssetTargetFallbacks.Count > 0);
```

Comment:
Count is always a better choice compare to Any() performance wise since it's a property compare to Any() which is an enumerable API.